cloud/cloud_manifest_test.cc

```diff
@@ -26,26 +26,24 @@ TEST_F(CloudManifestTest, BasicTest) {
   {
     std::unique_ptr<CloudManifest> manifest;
     ASSERT_OK(CloudManifest::CreateForEmptyDatabase("firstEpoch", &manifest));
-    manifest->Finalize();
 
-    ASSERT_EQ(manifest->GetEpoch(130).ToString(), "firstEpoch");
+    ASSERT_EQ(manifest->GetEpoch(130), "firstEpoch");
   }
   {
     std::unique_ptr<CloudManifest> manifest;
     ASSERT_OK(CloudManifest::CreateForEmptyDatabase("firstEpoch", &manifest));
     manifest->AddEpoch(10, "secondEpoch");
     manifest->AddEpoch(10, "thirdEpoch");
     manifest->AddEpoch(40, "fourthEpoch");
-    manifest->Finalize();
```

Comment:
How coma finalize is no longer needed?

Reply:
Currently, once db is opened, `CLOUDMANIFEST` won't be changed. But that's not the case for leader/follower anymore. In leader/follower, we will roll the epoch during leader election without reopening the DB.